be/src/exprs/bitmap_function.cpp

```diff
@@ -465,6 +465,44 @@ StringVal BitmapFunctions::bitmap_from_string(FunctionContext* ctx, const String
     return result;
 }
 
+BooleanVal BitmapFunctions::bitmap_contains(FunctionContext* ctx, const StringVal& src, const BigIntVal& input) {
+    if (src.is_null || input.is_null) {
+        return BooleanVal::null();
+    }
+
+    RoaringBitmap bitmap;
+    if (src.len == 0) {
+        bitmap.merge(*reinterpret_cast<RoaringBitmap*>(src.ptr));
```

Comment:
This operation is wasteful.
It is efficient to operate with src.ptr.

Reply:
ok